### PR TITLE
New Guns and Mosin Rework and Sawn PKA fix

### PR DESCRIPTION
**Pull request by VividPups**
## About the PR
adds the SVT40
add the Valentine 
add the Space Cleaner grenade
add the M34 'Willie Pete' grenade  (Kes request)  
add chicken nugget recipe
Rework the Mosin to allow Melee now 

## Why / Balance
Well, there aren't a lot of End shift Legal weapons, this PR adds a point between The Mosin and the AK with the new SVT40 rifle also added another pistol to be put inside the loot pool for gunships  (SVT could be in T3) 
As of now, I have added them to the less lethal Vending machine, and Liberation. Until the Loot spawn.yml is approve in the Research Bunker PR
## Technical details
Also, Clean up some of the Files for Texures and Weapons to be in there Respective parts 

## Media
image
image
image
image
image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

again just some File cleaning in the Weapons area

**Changelog**
- tweak: New sharpened Bayonets had outfitted the Mosins now 
- add: SVT40 and Valentine
- add:  Space Cleaner Grenade and M34 Willie Pete Grenade
- add: Chicken nuggets now have a cooking recipe.

### CODE REVIEW

**Comment by VividPups** [comment]
Yes, I know about  the File with the Revolver ( I will fix it) and Spawner file (I will learn how to remove it)

**Comment by VividPups** [comment]
Im not sure what the problem is with the Sprites.  They are just fine in-game for me

**Comment by VividPups** [comment]
Fix, the One for the gun, im going to need another set of eyes for this other error I'm getting, because I'm not seeing why it is saying that

**Review by TsjipTsjip — CHANGES_REQUESTED** [review]
Most item descriptions have language mistakes in them. I've went through and suggested changes for all of those which were directly apparent.

Your CL could be shortened into four entries, one for the mosin, one for the two new guns ("New guns: The SVT140 and Valentine pistol"), one for the two new grenades (see before) and one for the nugget recipe. I'd also reword this last one to not be roleplay-style: "Chicken nuggets now have a cooking recipe."

Lastly, and end-of-comment marker (`-->`) has snuck in below your CL entries.

File: Resources/Prototypes/_NF/Entities/Objects/Weapons/Ammunition/explosives.yml
Comment (VividPups) [reply Resources/Prototypes/_NF/Entities/Objects/Weapons/Ammunition/explosives.yml:44]: You aren't post to know the Arachnid is

File: Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Guns/pistols.yml
Comment (VividPups) [reply Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Guns/pistols.yml:5]: Just because it is old doesn't mean it still isn't mass produced,  IRL the Mosin is still in the active military still made and produced and still sold.  The AK is still active in military, still produce in mass.  both guns are extremely old, Yes S.T.A.R is an acronym it stands for  "Special Tactics and Rescue Service". Its a Resident Evil Reference

**Comment by VividPups** [comment]
@TsjipTsjip @GreaseMonk ,  I think I made all the changes.

**Comment by VividPups** [comment]
> It makes no sense to add guns and then also add a cooking recipe, in the same PR, so split them up please, you could even make 3, so people can decide to add guns and food, but not the grenade.
> 
> Got some more points of concern here:
> 
> * Please don't add napalm grenades to the gun vend that you can mass buy
> * Where did you get sprites from? If i can personally insert my opinion here, Im not super fond of the sprites and i cant put my finger on it to tell you what it is but its different
> * "Kesco and Sam's Weaponary Emporium" i think i read that KesCo is a company ingame, and there are posters referring to people, but there is a ship rule as well that you cant put your name on everything. Not my decision though
> * Remove `# For you Slips,` theyre bad comments
> 
> Please please make sure the guns are balanced, so it is ready for people to test and its not immediately overkill

-If I would to split up them in others PR I can 100% guarantee that something on my visual studios will break and it will take even more time to fix. I don't recall Dirv talking to me about the PR at all. 

- where do you want the napalm  grenades than, I originally wanted it in a Loot spawner.  It's almost been  months and a half  now since a Maintainer message me about it.

- Sam's weaponry emporium is the full name of the company. (Same how Kesco, has her name on it) 

- the napalm and space cleaner has the same Form of the Tear gas grenade.


I Tested the weapons to what I thought was fair. I suggest everytime I add a weapon or a event for the maintainers to test it

**Comment by GrandKadoer** [comment]
"No NFSD won't get warcrime grenades" but then they're in a vendor? These are class 3 contraband, and should probably be on expeds.

File: Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Guns/pistols.yml
Comment (TsjipTsjip) [inline Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Guns/pistols.yml:29]: Description can be omitted to parent off of lethal prototype.

File: Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Guns/pistols.yml
Comment (TsjipTsjip) [inline Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Guns/pistols.yml:5]: "Uses .35 auto ammo.", for parallel with Mosin above.

File: Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Guns/snipers.yml
Comment (TsjipTsjip) [inline Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Guns/snipers.yml:18]: .30 Cal is not a bullet type, this should match the name of the round this is supposed to fit exactly as is shown in-game. (Check mosin for an example.) I think this takes `.30 rifle` ammo, but I can't tell from here.

Treason should not be capitalised unless it's a stylistic choice. You're missing a period at the end of the description.

**Review by TsjipTsjip — CHANGES_REQUESTED** [review]
Some other minor things on the updated version. Nearly good tho.

**Comment by dvir001** [comment]
https://github.com/space-wizards/space-station-14/pull/25444

Omit cleaner.

**Comment by dvir001** [comment]
Also your mosins made my build crash none stop when trying to use them now. (Build mod, not debug)

**Comment by VividPups** [comment]
> Also your mosins made my build crash none stop when trying to use them now. (Build mod, not debug)

you know more Info then it crashes would help, Espically since I don't have "Build" and the Video is showing us to use Release

**Comment by VividPups** [comment]
> > > Also your mosins made my build crash none stop when trying to use them now. (Build mod, not debug)
> > 
> > 
> > you know more Info then it crashes would help, Espically since I don't have "Build" and the Video is showing us to use Release
> 
> It crashes when a mosin is handed out of a vending machine on frontier i think

I tested it on both vending machines that has it and spawn new ones. In and it isn't crashing or showing any errors for me.   I put screen shots on the development chat

**Comment by VividPups** [comment]
I'm going to re-PR this at a later date. I however will not close until the new on is up.  I will be putting the Mosin by itself,  the SVT and Valentine together, and the Grenades In a different on.

**Pull request closed without merging**
